CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Simplify CLI option handling by only allowing one option at a time to be added ([#54])
 - Redefine OQT regions ([#26])
 - Use ohsome API endpoint `/contributions/latest/count` for Last Edit indicator ([#68])
+- Change type of attribute bpolys to be GeoJSON.Feature ([#69] [#9])
```

Comment:
This change requires a **breaking** change. Please sort this accordingly.

Reply:
Added it to Breaking Changes section.